aptos-move/framework/aptos-stdlib/sources/cryptography/bls12381.move

```diff
@@ -330,369 +389,126 @@ module aptos_std::bls12381 {
         message: vector<u8>
     ): bool;
 
-    //
-    // Constants and helpers for tests
-    //
+    public fun public_key_with_pop_to_unvalidated(pkpop: &PublicKeyWithPoP): PublicKey {
```

Comment:
You need to add comments here, explaining that this "demotes" a `PublicKeyWithPoP` to a `PublicKey` so that it can be used in functions that accept a `PublicKey` (perhaps give example?).